main_page.php

```diff
@@ -0,0 +1,30 @@
+<?php
+$db = new PDO('mysql:host=db; dbname=collectorapp', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
```

Comment:
You could pop this into a function, that way on any future pages you can just call the function to get a new DB connection!